Modules/Core/Common/test/itkImageRangeGTest.cxx

```diff
@@ -37,6 +37,7 @@ template class itk::Experimental::ImageRange<const itk::Image<short>>;
 template class itk::Experimental::ImageRange<itk::VectorImage<short>>;
 
 using itk::Experimental::ImageRange;
+using itk::Experimental::MakeImageRange;
```

Comment:
Are you specifying each used member on purpose? As opposed to just `using itk::Experimental`?

Reply:
@dzenan In general, I'm reluctant to use `using namespace X`, as it might put the whole purpose of namespaces at risk. Doing `using X::A` and `using X::B` appears more careful to me. And as this GTest file is specifically about testing `ImageRange` and `MakeImageRange`, I feel that it's entirely appropriate to add `using`'s for these two members.

However... if you would very much prefer `using namespace itk::Experimental` in this case, I wouldn't mind amending the commit accordingly  :-) Please tell me.

Reply:
If you are doing it on purpose, that's fine 😄